images/atlantis/config/main.tf

```diff
@@ -0,0 +1,38 @@
+variable "extra_packages" {
+  description = "The additional packages to install"
+  type        = list(string)
+  default     = []
+}
+
+variable "packages" {
+  # https://github.com/runatlantis/atlantis/blob/main/Dockerfile#L44
+  description = "List of packages as runtime dependencies for Atlantis"
+  type        = list(string)
+  default = [
+    "atlantis",
+    "busybox",
+    "conftest",
+    "curl",
+    "git",
+    "git-lfs",
+    "openssh-server",
+    "libcap",
+    "dumb-init",
+    "gnupg",
+    "openssl",
+  ]
+}
+
+module "accts" { source = "../../../tflib/accts" }
+
+output "config" {
```

Comment:
I previously had this included in the config...
```
environment = {
  # https://github.com/runatlantis/atlantis/blob/main/Dockerfile#L7
  DEFAULT_CONFTEST_VERSION = "0.46.0"
}
```
...but I removed it because we cannot dynamically set the version to the latest of the wolfi `conftest` package through Terraform. Atlantis should be able to [pick up the local version](https://github.com/runatlantis/atlantis/blob/main/server/core/runtime/policy/conftest_client.go#L134), but if all fails, it will attempt to download the binary from source. What this means is the end user will see this warning log when they run the image, as the `DEFAULT_CONFTEST_VERSION` variable never gets set by us:

```
{"level":"info","ts":"2023-10-23T19:08:04.261Z","caller":"policy/conftest_client.go:151","msg":"failed to get default conftest version. Will attempt request scoped lazy loads DEFAULT_CONFTEST_VERSION not set","json":{}}
```